internal/pkg/cli/env_init.go

```diff
@@ -206,9 +206,25 @@ func (o *initEnvOpts) Validate() error {
 			return err
 		}
 	}
+
 	if o.AppName() == "" {
-		return fmt.Errorf("no application found: run %s or %s into your workspace please", color.HighlightCode("app init"), color.HighlightCode("cd"))
+		existingApps, _ := o.store.ListApplications()
+		if len(existingApps) == 0 {
+			o.askAppName(fmtAppInitNamePrompt)
+		} else {
+			useExistingApp, err := o.prompt.Confirm(
+			"Would you like to use one of your existing applications?", "", prompt.WithTrueDefault(), prompt.WithFinalMessage("Use existing application:"))
```

Comment:
I think we should probably just prompt them to select the existing application. If there's no application on the list that they'd like to use, it means that the application they'd like to use doesn't exist yet and they should `app init` to create a new one.